packages/mask-sdk/main/wallet.ts

```diff
@@ -27,7 +27,7 @@ class EthereumEventEmitter extends EventTarget implements Mask.Ethereum.MaskEthe
     }
 }
 class MaskProvider extends EthereumEventEmitter implements Mask.Ethereum.ProviderObject {
-    async request(param: unknown) {
+    async request(param: any): Promise<any> {
         return contentScript.eth_request(param)
     }
```

Comment:
```suggestion
    async request<T>(param: RequestArguments): Promise<T> {
        return contentScript.eth_request(param)
    }
```

Reply:
this function implements an overloaded function, therefore it cannot be typed this way. The consumer won't see this `any` signature because it's purely internal. We'll publish a `.d.ts` package that contains the definition of `Mask` namespace.